dotCMS/src/main/webapp/dotcms-webcomponents/p-79c65723.system.js

```diff
@@ -0,0 +1 @@
gName$);{n.$flags$|=8}try{new s(n)}catch(v){_(v)}{n.$flags$&=~8}{n.$flags$|=128}i();o=Y(a.$tagName$);if(!(!L.has(o)&&s.style))return[3,5];l=D("registerStyles",a.$tagName$);f=s.style;if(!(a.$flags$&8))return[3,4];return[4,r.import("./p-6cef36c5.system.js").then((function(e){return e.scopeCss(f,o,false)}))];case 3:f=c.sent();c.label=4;case 4:J(o,f,!!(a.$flags$&1));l();c.label=5;case 5:$=n.$ancestorComponent$;u=function(){return Re(e,n,a,true)};if($&&$["s-rc"]){$["s-rc"].push(u)}else{u()}return[2]}}))}))};var Me=function(e,r){if((m.$flags$&1)===0){var t=D("connectedCallback",r.$tagName$);var n=w(e);if(r.$listeners$){n.$rmListeners$=Ce(e,n,r.$listeners$)}if(!(n.$flags$&1)){n.$flags$|=1;{if(r.$flags$&4||r.$flags$&8){qe(e)}}{var a=e;while(a=a.parentNode||a.host){if(a["s-p"]){we(n,n.$ancestorComponent$=a);break}}}if(r.$members$){Object.entries(r.$members$).forEach((function(r){var t=r[0],n=r[1][0];if(n&31&&e.hasOwnProperty(t)){var a=e[t];delete e[t];e[t]=a}}))}{I((function(){return Be(e,n,r)}))}}t()}};var qe=function(e){var r=e["s-cr"]=h.createComment("");r["s-cn"]=true;e.insertBefore(r,e.firstChild)};var ze=function(e){if((m.$flags$&1)===0){var r=w(e);{if(r.$rmListeners$){r.$rmListeners$();r.$rmListeners$=undefined}}if(m.$cssShim$){m.$cssShim$.removeHost(e)}}};var He=e("b",(function(e,r){if(r===void 0){r={}}var t=D();var n=[];var a=r.exclude||[];var i=v.customElements;var s=h.head;var o=s.querySelector("meta[charset]");var l=h.createElement("style");var f=[];var $;var u=true;Object.assign(m,r);m.$resourcesUrl$=new URL(r.resourcesUrl||"./",h.baseURI).href;if(r.syncQueue){m.$flags$|=4}e.forEach((function(e){return e[1].forEach((function(r){var t={$flags$:r[0],$tagName$:r[1],$members$:r[2],$listeners$:r[3]};{t.$members$=r[2]}{t.$listeners$=r[3]}{t.$attrsToReflect$=[]}{t.$watchers$={}}if(!p&&t.$flags$&1){t.$flags$|=8}var s=t.$tagName$;var o=function(e){__extends(r,e);function r(r){var n=e.call(this,r)||this;r=n;N(r);if(t.$flags$&1){if(p){{r.attachShadow({mode:"open"})}}else if(!("shadowRoot"in r)){r.shadowRoot=r}}return n}r.prototype.connectedCallback=function(){var e=this;if($){clearTimeout($);$=null}if(u){f.push(this)}else{m.jmp((function(){return Me(e,t)}))}};r.prototype.disconnectedCallback=function(){var e=this;m.jmp((function(){return ze(e)}))};r.prototype.forceUpdate=function(){xe(this,t)};r.prototype.componentOnReady=function(){return w(this).$onReadyPromise$};return r}(HTMLElement);t.$lazyBundleIds$=e[0];if(!a.includes(s)&&!i.get(s)){n.push(s);i.define(s,Ue(o,t,1))}}))}));{l.innerHTML=n+W;l.setAttribute("data-styles","");s.insertBefore(l,o?o.nextSibling:s.firstChild)}u=false;if(f.length>0){f.forEach((function(e){return e.connectedCallback()}))}else{{m.jmp((function(){return $=setTimeout(Ee,30)}))}}t()}));var Ve=e("c",(function(e,r,t){var n=Fe(e);return{emit:function(e){var a=new CustomEvent(r,{bubbles:!!(t&4),composed:!!(t&2),cancelable:!!(t&1),detail:e});n.dispatchEvent(a);return a}}}));var Fe=e("g",(function(e){return w(e).$hostElement$}))}}}));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The for-in loop variable 'n' should be explicitly scoped with var to avoid pollution.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5062311)